src/api/dataStoreApi/actions/getSubdomainsById.ts

```diff
@@ -26,6 +27,9 @@ export const getSubdomainsById = async (
     if (sortBy && sortDirections) {
       requestUri += `&${sortBy}&${sortDirections}`;
     }
+    if (nameFilter) {
```

Comment:
Should we check to make sure nameFilter is not an empty string also?

also, possibly, might want to encode it so you dont bork the URI with special characters  like &

Reply:
Good callout, but a nice javascript feature is that the compilter will actually check for `""` on this check and return false when it detects the variable as type `string`

To your second pt I have verified that the only allowed special character (.) works, but let's have a follow up discussion on if we need to implement encoding in the search endpoints. (Not sure the effect encoding would as all endpoints using subgraph and datastore don't use it)